apiserver/facades/client/application/privileges.go

```diff
@@ -0,0 +1,42 @@
+package application
+
+import (
+	"github.com/juju/errors"
+	"gopkg.in/juju/environschema.v1"
+)
+
+const privilegeConfigOptionName = "elevated_privileges"
+
+var privilegeFields = environschema.Fields{
+	privilegeConfigOptionName: {
+		Description: "blah blah",
+		Type:        environschema.Tbool,
+		Group:       environschema.JujuGroup,
+	},
+}
+
+//PrivilegeSchema builder pattern which returns an existing schema augmented
+//with the application privilege schema fields.
+func PrivilegeSchema(providerFields environschema.Fields) (environschema.Fields, error) {
```

Comment:
Why have PrivilegeSchema and addPrivilegeSchema where one only exists to call the other one.

Is there an intent to do more than just addPr... here?
If not, then I would just say move the code of addPrivilege into this function, and update callers to call the single function.